libraries/AP_AccelCal/AccelCalibrator.cpp

```diff
@@ -379,9 +379,6 @@ void AccelCalibrator::run_fit(uint8_t max_iterations, float& fitness)
         }
 
         num_iterations++;
-        if (fitness - last_fitness < 1.0e-9f) {
-            break;
-        }
```

Comment:
do you think it makes sense to stop iterating if the value isn't changing? Or do you prefer to keep iterating?